MekHQ/src/mekhq/gui/panels/StartupScreenPanel.java

```diff
@@ -41,22 +42,47 @@
 import java.awt.event.WindowEvent;
 import java.awt.image.BufferedImage;
 import java.io.File;
+import java.io.FilenameFilter;
+import java.util.Arrays;
+import java.util.List;
 
 public class StartupScreenPanel extends AbstractMHQPanel {
     //region Variable Declarations
     private MekHQ app;
     private File lastSaveFile;
     private BufferedImage backgroundIcon;
+
+    // Save file filtering needs to avoid loading some special files
+    static public FilenameFilter saveFilter = new FilenameFilter() {
+        @Override
+        public boolean accept(File dir, String name) {
+            // Allow any .xml, .cpnx, and .cpnx.gz file that is not in the list of excluded files
+            boolean accepted = false;
+            List<String> toReject = Arrays.asList(
+                    PreferenceManager.DEFAULT_CFG_FILE_NAME.toLowerCase()
+            );
+            accepted = (
```

Comment:
Not sure what to make of this artsy formatting here. This could be written in two lines and "accepted" could be inlined.

Reply:
Speaking of lines 64++